### DISCUSSION TITLE
BaseURL is appending a "/" when I don't want it

### DISCUSSION THREAD

**Discussion by CodyBontecou** [cypress-io/cypress#18173] [Questions and Help]
I'm trying to organize my storybook + cypress tests and within my config file have my baseURL set like:

`"baseUrl": "http://localhost:3003/iframe.html?id=",`

and within the test I visit URL's like so:

`cy.visit('avatar--user-with-img')`

Unfortunately, when I run my test, it loads the following URL into my cypress test runner:

`http://localhost:3003/iframe.html?id=/avatar--user-no-img`

Is there a config variable somewhere that would allow me to remove this slash from being generated?

**Comment by jjhelguero** [comment]
You may want to update your `"baseUrl"` to either user `http://localhost:3003` or `http://localhost:3003/iframe.html`.
